www/install/sql/centreon/Update-DB-20.04.5.sql

```diff
@@ -4,6 +4,7 @@ CREATE TABLE `user_filter` (
     `user_id` int(11) NOT NULL,
     `page_name` varchar(255) NOT NULL,
     `criterias` text,
+    `order` int(11) NOT NULL,
```

Comment:
wasn't the 20.04.5 already released ?

Reply:
If order cannot be NULL and if the script should be upgraded to 20.04.6 : 
Then, what will happen for already created filters ?